### ISSUE TITLE
[webview_flutter] Bug with scrollView and WebView on iOS

### ISSUE THREAD

**Issue by robotoss** [flutter/flutter#107639]
## Steps to Reproduce

1. Execute `flutter create new project` on the code sample 
2. Add ListView to body with WebView and some Containers
3. Scroll

**Actual results:
The webview widget stops at the safeArea, and after a certain period of time it starts scrolling forward

https://user-images.githubusercontent.com/34773197/178974877-630c28fe-6f3d-4da8-8bc1-3c64e1877522.mov

<details>
<summary>Code sample</summary>

```dart
ListView(
        children: [
          Container(
            height: 290,
            width: double.infinity,
            color: Colors.red,
          ),
          Container(
            color: Colors.green,
            height: 290,
            child: const WebView(
              initialUrl: 'https://google.com/',
            ),
          ),
          Container(
            height: 290,
            width: double.infinity,
            color: Colors.red,
          ),
          Container(
            color: Colors.green,
            height: 290,
            child: const WebView(
              initialUrl: 'https://google.com/',
            ),
          ),
          Container(
            height: 290,
            width: double.infinity,
            color: Colors.red,
          ),
          Container(
            color: Colors.green,
            height: 290,
            child: const WebView(
              initialUrl: 'https://google.com/',
            ),
          ),
          Container(
            height: 700,
            width: double.infinity,
            color: Colors.green,
          ),
        ],
      )
```

</details>

```
[✓] Flutter (Channel stable, 3.0.4, on macOS 12.4 21F79 darwin-arm, locale en-KZ)
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 13.4.1)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2021.2)
[✓] IntelliJ IDEA Community Edition (version 2022.1.3)
[✓] VS Code (version 1.69.1)
[✓] Connected device (3 available)
[✓] HTTP Host Availability
```

webview_flutter: 3.0.4

**Comment by robotoss** [comment]
`self.scrollView.contentInsetAdjustmentBehavior = UIScrollViewContentInsetAdjustmentNever; white_check_mark eyes raised_hands`

https://github.com/flutter/plugins/pull/2343/commits/5e80a470bdb2cbd444d9c4e95ec69f27e6ad3c38

Doesn't work because for some reason this part is missing in the current version of the plugin

**Comment by maheshmnj** [comment]
Hi @robotoss, Thanks for filing the issue. What device is this issue on? I tried to reproduce the issue on iPhone 12 pro simulator (IOS 14.5) it just lags when the webview goes out / comes into view. But the scroll remains in sync.


https://user-images.githubusercontent.com/31410839/178995651-de7ddf96-4dd6-4bbb-b0ea-e678d6ae47f0.mov

<details>
<summary>flutter doctor -v</summary>

```
[✓] Flutter (Channel stable, 3.0.5, on macOS 12.4 21F79 darwin-arm, locale en-IN)
    • Flutter version 3.0.5 at /Users/mahesh/Documents/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision f1875d570e (19 hours ago), 2022-07-13 11:24:16 -0700
    • Engine revision e85ea0e79c
    • Dart version 2.17.6
    • DevTools version 2.12.2

[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0-rc4)
    • Android SDK at /Users/mahesh/Library/Android/sdk
    • Platform android-32, build-tools 33.0.0-rc4
    • ANDROID_HOME = /Users/mahesh/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.2.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2021.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)

[✓] IntelliJ IDEA Community Edition (version 2021.2.1)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 61.2.4
    • Dart plugin version 212.5080.8

[✓] VS Code (version 1.67.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.44.0

[✓] Connected device (3 available)
    • iPhone 12 Pro (mobile) • C217DCBE-806E-4BC7-A952-61E4A647A7DA • ios            •
      com.apple.CoreSimulator.SimRuntime.iOS-14-5 (simulator)
    • macOS (desktop)        • macos                                • darwin-arm64   • macOS 12.4 21F79 darwin-arm
    • Chrome (web)           • chrome                               • web-javascript • Google Chrome 103.0.5060.114

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```
</details>

**Comment by robotoss** [comment]
iPhone 23 pro max simulator (IOS 15.5)

<img width="463" alt="Screen Shot 2022-07-14 at 16 42 41" src="https://user-images.githubusercontent.com/34773197/178996449-e8f76552-eba7-464c-8354-6b299a9d059c.png">

This problem appears on several working devices
Mac pro m1 pro - Monterey / Emulator iOS(15.5)
Mac pro m1 - Monterey / Emulator iOS(15.2)
Mac pro i5 Big Sur / Emulator iOS(15.5)
Mac pro i5 Big Sur / Emulator iOS(15.2)

**Comment by bparrishMines** [comment]
Fixed in `2.9.1` and https://github.com/flutter/plugins/pull/6104
